src/app/main-navigation/main-navigation.component.html

```diff
@@ -8,14 +8,22 @@
   <div class="collapse navbar-collapse" id="navbarMain" [ngbCollapse]="isNavbarCollapsed">
     <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
       <li class="nav-item" *ngFor="let link of primaryNavigationLinks" routerLinkActive="active">
-        <a class="nav-link" routerLink="{{ link.route }}">{{ link.name}}</a>
+        <a class="nav-link" routerLink="{{ link.route }}">{{ link.name | translate }}</a>
+      </li>
+    </ul>
+    <ul class="navbar-nav mr-5 mt-2 mt-lg-0">
```

Comment:
Here are the language selection buttons in the navbar where user is able to change display language